_data/developer.yml

```diff
@@ -423,6 +423,18 @@ websites:
   btc: true
   othercrypto: true
   doc: https://ultimez.com/blog/ultimez-now-accepts-cryptocurrency-payment-web-services/
+- name: Underlabs Inc.
+  url: https://underlabs.ca
+  img: underlabs.png
+  twitter: underlabs
+  email_address: hello@underlabs.ca
+  region: na
+  bch: Yes
+  btc: Yes
+  othercrypto: No
+  doc: https://underlabs.ca/smart-contracts.html
```

Comment:
That is not a good doc link as it does not confirm if this service is accepting BCH